src/types.ts

```diff
@@ -87,6 +87,8 @@ export interface InterventionProps {
   length_meters?: number;
   // For LineStrings and Polygons only
   waypoints?: Waypoint[];
+  //For polygons only
+  is_coverage_polygon: boolean;
```

Comment:
I don't think I can make this optional because I don't think I can ts-ignore the warning when it's assigned to a checkbox in a form

Reply:
Hmm... this makes previous files incompatible. I think it _can_ be optional, and in the pipeline form, you can see if it's undef and make it be false before the checkbox